examples/complete-alb/main.tf

```diff
@@ -576,20 +500,30 @@ data "aws_route53_zone" "this" {
 
 module "acm" {
   source  = "terraform-aws-modules/acm/aws"
-  version = "~> 4.0"
+  version = "~> 3.0"
```

Comment:
```suggestion
  version = "~> 4.0"
```

Reply:
fixed in https://github.com/terraform-aws-modules/terraform-aws-alb/pull/305/commits/522c07009fa3cc79200886990f3ab8ba1ac76231